aep/general/0133/aep.md.j2

```diff
@@ -0,0 +1,246 @@
+  - The collection identifier (`books` in the above example) **must** be a
+    literal string.
+- There **must** be a `body` key in the `google.api.http` annotation, and it
+  **must** map to the resource field in the request message.
+  - All remaining fields **should** map to URI query parameters.
+- There **should** be exactly one `google.api.method_signature` annotation,
+  with a value of `"parent,{resource},id"`, or "`"parent,{resource}"` if the
+  resource ID is not required.
+
+{% tab oas %}
+
+**Note:** OAS guidance not yet written
+
+{% endtabs %}
+
+### Request message
+
+Create methods implement a common request message pattern:
+
+- An `id` field **must** be included for management plane resources, and
+  **should** be included for data plane resources.
+- The resource field **must** be included and **must** map to the POST body.
+- The request message **must not** contain any other required fields and
+  **should not** contain other optional fields except those described in this
+  or another AEP.
+
+{% tab proto %}
+
+```proto
+message CreateBookRequest {
+  // The parent resource where this book will be created.
+  // Format: publishers/{publisher}
+  string parent = 1 [
+    (google.api.field_behavior) = REQUIRED,
+    (google.api.resource_reference) = {
+      child_type: "library.example.com/Book"
+    }];
+
+  // The ID to use for the book, which will become the final component of
+  // the book's resource path.
+  //
+  // This value should be 4-63 characters, and valid characters
+  // are /[a-z][0-9]-/.
+  string id = 2 [(google.api.field_behavior) = REQUIRED];
+
+  // The book to create.
+  Book book = 3 [(google.api.field_behavior) = REQUIRED];
+}
+```
+
+- A `parent` field **must** be included unless the resource being created is a
+  top-level resource. It **should** be called `parent`.
+  - The field **should** be [annotated as `REQUIRED`][aep-203].
+  - The field **must** identify the [resource type][aep-123] of the resource
+    being created.
+
+{% tab oas %}
+
+**Note:** OAS guidance not yet written
+
+{% endtabs %}
+
+### Long-running create
+
+Some resources take longer to create a resource than is reasonable for a
+regular API request. In this situation, the API **should** use a long-running
+operation (AEP-151) instead:
+
+- The response type **must** be set to the resource (what the return type would
+  be if the RPC was not long-running).
+
+{% tab proto %}
+
+```proto
+rpc CreateBook(CreateBookRequest) returns (aep.api.Operation) {
+  option (google.api.http) = {
+    post: "/v1/{parent=publishers/*}/books"
+  };
+  option (aep.api.operation_info) = {
+    response_type: "Book"
+    metadata_type: "OperationMetadata"
+  };
+}
+```
+
+- Both the `response_type` and `metadata_type` fields **must** be specified.
+
+{% tab oas %}
+
+**Note:** OAS guidance not yet written
+
+{% endtabs %}
+
+**Important:** Declarative-friendly resources (AEP-128) **should** use
```

Comment:
we can remove this guidance - We were working toward removing AIP-128 from aip.dev anyway, in lieu of just adding guidance in the rest of the AIPs.

Reply:
Removed.